### ISSUE TITLE
proxy

### ISSUE THREAD

**Issue by torloneg** [api-ai/api-ai-node-js#28]
It should be exposed the request properties to set the proxy

**Comment by telkomops** [comment]
Agree. Came here to create an issue. SDK should support working through proxies.

**Comment by androidfanatic** [comment]
+1, there needs to be support for proxy configuration preferably controlled via environment variables.

**Comment by reyou** [comment]
I was expecting to work with fiddler, but no luck.
https://weblogs.asp.net/dixin/use-fiddler-with-node-js

Please add this as high priority.
